src/test/java/steps/hooks.java

```diff
@@ -4,24 +4,31 @@
 import cucumber.api.java.Before;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
 
 public class hooks {
 
     public static WebDriver driver;
 
     @Before
     public void beforeScenarios() {
-        System.setProperty("webdriver.chrome.driver", "/Users/ambichaliki/Documents/automation/src/main/resources/chromedriver");
-        driver = new ChromeDriver();
-        driver.get("https://login.salesforce.com/?locale=uk");
-        System.out.println("Before Scenario-Navigated to salesforce login page successfully ");
+       System.setProperty("webdriver.chrome.driver", "/Users/ambichaliki/Documents/automation/src/main/resources/chromedriver");
+       driver = new ChromeDriver();
+        //driver.get("https://login.salesforce.com/?locale=uk");
+        //driver.get("https://www.eenadu.net/");
+        driver.get("https://www.phptravels.net");
+
+//        System.setProperty("webdriver.gecko.driver", "/Users/ambichaliki/Documents/automation/src/main/resources/geckodriver");
+//         driver = new FirefoxDriver();
+//         driver.get("https://eenadu.net");
+//        System.out.println("Navigated to Eenadu successfully  ");
```

Comment:
Please remove this unwanted commented out code.
If you think this is useful for your reference move all these into a separate branch or save somewhere else.
So that we will have clean tests focus on what we need and eliminate the confusion.
Initially, its ok to comment out bits of code but later stages we cannot manage and we forgot why we comment this code.